### DISCUSSION TITLE
Anonymising JPEG or other embedded tags

### DISCUSSION THREAD

**Discussion by jas88** [SMI/SmiServices#952] [Ideas]
Since the DICOM file usually contains an embedded JPEG or TIFF file as the "pixel data", we should probably extract and examine the metadata within that as we already do for DICOM tags and strip/redact as appropriate? It won't be directly patient-specific, but things like timestamps could be an issue we'd care about, particularly if we're rounding off the timestamps in the DICOM tags.

**Comment by rkm** [comment]
Great idea.

It looks like we don't extract this info at the moment unfortunately, since fo-dicom stores this in`FileMetaInfo` rather than the `Dataset` object. So we can implement this and it will be stored for any new files we scan, but going back and retroactively adding it to the NPACS data will take some effort...

![transfersyntax](https://user-images.githubusercontent.com/5160559/140317291-1b2de161-fc46-4921-b595-1128c83db3b2.PNG)

**Comment by jas88** [comment]
Maybe something to check for during the extraction/redaction pipeline rather than extracting beforehand - I don't think there's anything we'd want to index/promote for cohort building, just check we aren't leaking anything unwanted through the anon pipeline?
